README.md

```diff
@@ -18,14 +18,15 @@ cloud-provider-azure is still under **alpha** stage and its releases are maintai
 
 The latest version of azure-cloud-controller-manager and azure-cloud-node-manager could be found at
 
-* `mcr.microsoft.com/k8s/core/azure-cloud-controller-manager:v0.3.0`
-* `mcr.microsoft.com/k8s/core/azure-cloud-node-manager:v0.3.0`
+* `mcr.microsoft.com/k8s/core/azure-cloud-controller-manager:v0.4.0`
+* `mcr.microsoft.com/k8s/core/azure-cloud-node-manager:v0.4.0`
```

Comment:
those images haven't been pushed yet, let's wait a while for pipeline success
/hold